src/parameterized_tendencies/microphysics/precipitation.jl

```diff
@@ -211,6 +211,7 @@ function compute_precipitation_cache!(Y, p, colidx, ::Microphysics1Moment, _)
     α(ts) = TD.Parameters.cv_l(thp) / Lf(ts) * (Tₐ(ts) - cmp.ps.T_freeze)
 
     qₚ(ρqₚ, ρ) = max(FT(0), ρqₚ / ρ)
+    limit(q, dt) = q / dt
```

Comment:
Could you please add a comment about the limiter application in the docs section under `Microphysics source terms` so we can track it against future changes ? (Presumably when we have better approximations the intent is to relax or update these limiters?)